src/Card.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Card.css';
+
+const Card = ({poster, title, releaseDate, id, rating, showSingleMovie}) => {
+  return (
+    <div onClick={() => showSingleMovie(id)}>
+      <img className='all-movie-posters' src={poster} alt='movie poster'/>
```

Comment:
How can you utilize the data you have to create a more detailed alt description for people using screen readers?